models/intel/asl-recognition-0004/description/asl-recognition-0004.md

```diff
@@ -18,23 +18,24 @@ on the input clip.
 
 | Metric                          | Value                                     |
 |---------------------------------|-------------------------------------------|
-| Top-1 accuracy (MS-ASL-100\*)   | 0.847                                     |
+| Top-1 accuracy (MS-ASL-100)     | 0.847                                     |
 | GFlops                          | 6.660                                     |
 | MParams                         | 4.133                                     |
 | Source framework                | PyTorch\*                                 |
 
 ## Inputs
 
-Name: `input`, shape: [1x3x16x224x224]. An input image sequence in the format [BxCxTxHxW], where:
- - B - batch size
- - C - number of channels
- - T - duration of input clip
- - H - image height
- - W - image width
+Name: `input`, shape: `1, 3, 16, 224, 224`. An input image sequence in the format `B, C, T, H, W`, where:
+
+ - `B` - batch size
+ - `C` - number of channels
+ - `T` - duration of input clip
+ - `H` - image height
+ - `W` - image width
 
 ## Outputs
 
-The model outputs a tensor with the shape [Bx100], each row is a logits vector of performed ASL gestures.
+The model outputs a tensor with the shape `1, 100`, each row is a logits vector of performed ASL gestures.
```

Comment:
I think here should be description like: 
> shape: `1, 100`, format `B, L`

Reply:
Done